apps/music/js/metadata/core.js

```diff
@@ -99,7 +99,7 @@ var AudioMetadata = (function() {
     // the appropriate parser.  Start off with an 64kb chunk of data.
     // If the metadata is in that initial chunk we won't have to read again.
     return new Promise(function(resolve, reject) {
-      var headersize = Math.min(64 * 1024, blob.size);
+      var headersize = Math.min(256 * 1024, blob.size);
```

Comment:
Why is this change necessary? (And if it is, please update the comment just before this.)